theme/src/components/hero.tsx

```diff
@@ -8,21 +9,18 @@ import { ImageSharpFluid } from '../types/graphql-types';
 interface Props {
   src: ImageSharpFluid;
   url?: string;
+  alt?: string;
 }
 
-export default function Hero({ src, url }: Props) {
+export default function Hero({ src, url, alt }: Props) {
   if (!src) return null;
 
   return (
     <HeroStyled>
       <Spacer size={1} />
-      {url ? (
-        <Link to={url}>
-          <Img sizes={src} />
-        </Link>
-      ) : (
-        <Img sizes={src} />
-      )}
+      <ConditionalWrap condition={Boolean(url)} wrap={children => <Link to={url!}>{children}</Link>}>
```

Comment:
Can you please remove `ConditionalWrap` ? 
I find those abstractions actually harder to read. 
If you are worried about duplication just create function for that Img tag :)

Reply:
Done. :)